domains/debajyoti.json

```diff
@@ -5,6 +5,6 @@
     },
 
     "record": {
-        "CNAME": "debajyoti.cf"
+        "A": ["152.67.6.15"]
```

Comment:
404 IPv4. Please provide a valid IPv4.

Reply:
This was a multi host ip...so moved the website to ip access.